### ISSUE TITLE
PriorGrad validation filelist

### ISSUE THREAD

**Issue by vvvm23** [microsoft/NeuralSpeech#89]
Greetings,

Why is the list of files using in the validation split so small? Won't this make validation loss quite noisy during training?

Furthermore, was the validation or test split used to produce the results in Figure 4 of the paper?

Thanks!

**Comment by L0SG** [comment]
Hi, thank you for checking out our work and sorry for a late reply.

We used a relatively small number of clips for validation as we have monitored training convergence using a full reverse diffusion schedule. Becuase sampling is very slow for the full schedule, we minimized the validation set to reduce the overhead. 

The diffusion model training is stable (compared to, say, GANs), so we find that the small numbers were good enough with low variance for this purpose. The figure 4 is obtained from this monitoring over small intervals of the training steps followed by smoothing. Other results are obtained from the test split.

**Comment by vvvm23** [comment]
Thank you!
